client/src/common/components/Home.tsx

```diff
@@ -1,8 +1,149 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+
+import {
+	Accordion,
+	AccordionItem,
+	AccordionItemHeading,
+	AccordionItemButton,
+	AccordionItemPanel,
+} from 'react-accessible-accordion';
+import 'react-accessible-accordion/dist/fancy-example.css';
 
 const Home = () => {
 	// test, added style here just to see the classes work
-	return <div　className="bg-primary_bg_color w-screen h-96 mt-headerHeight text-primary_title_color">This is Home</div>;
+	return (
+		<div className="bg-primary_bg_color text-primary_title_color bg-gradient">
+			<div className="p-10 md:p-20">
+				<div className="md:float-right w:20 md:w-2/4 ">
+					<img src="./assets/shared_workspace.svg" />
+				</div>
+
+				<div className="text-3xl font-black md:text-7xl mt-5 text-center md:text-justify">
+					Senpai Kohai
+				</div>
```

Comment:
```suggestion
				<h1 className="text-3xl font-black md:text-7xl mt-5 text-center md:text-justify">
					Senpai Kohai
				</h1>
```
Use `h1` instead because it's the title.